powersimdata/input/export_data.py

```diff
@@ -140,3 +149,281 @@ def export_transformed_profile(kind, scenario_info, grid, ct, filepath):
     profile = tp.get_profile(kind)
     print(f"Writing scaled {kind} profile to {filepath} on local machine")
     profile.to_csv(filepath)
+
+
+def export_to_pypsa(scenario, preserve_all_columns=False):
```

Comment:
Can we change the call signature to something more intuitive? Something like:
```
export_to_pypsa(scenario_or_grid, preserve_all_columns=False)
export_to_pypsa(scenario=None, Grid=None, preserve_all_columns=False)
export_to_pypsa(*, scenario=None, Grid=None, preserve_all_columns=False)
```
Having an input named `scenario` that can equivalently take a `Grid` object seems handy, but potentially confusing.